libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFSecurityLockout.m

```diff
@@ -520,7 +520,9 @@ + (void)presentPasscodeController:(SFPasscodeControllerMode)modeValue passcodeCo
         UIViewController *passcodeViewController = passcodeVcCreationBlock(modeValue, configData);
         [SFSecurityLockout setPasscodeViewController:passcodeViewController];
         SFPasscodeViewControllerPresentationBlock presentBlock = [SFSecurityLockout presentPasscodeViewControllerBlock];
-        presentBlock(passcodeViewController);
+        if (presentBlock != nil) {
```

Comment:
Should never be nil. But we are crashing with sigsev:

0   SalesforceSDKCore                    0x0000000101f13774 +[SFSecurityLockout presentPasscodeController:passcodeConfig:] (SFSecurityLockout.m:523)
1   SalesforceSDKCore                    0x0000000101f13458 +[SFSecurityLockout lock] (SFSecurityLockout.m:456)
2   SalesforceSDKCore                    0x0000000101f3e620 -[SalesforceSDKManager handleAppForeground:] (SalesforceSDKManager.m:435)